Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -861,6 +865,10 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
     // to ensure it's created with the right size.
     m_page->setDeviceScaleFactor(parameters.deviceScaleFactor);
 
+#if USE(SKIA)
+    FontRenderOptions::singleton().setUseSubpixelPositioning(parameters.deviceScaleFactor <= 2.);
```

Comment:
I think `2.0` definitely needs to be considered HiDPI, so this should compare with `< 2.0`, isn't it?

Generally speaking, I wonder from which scale factor onwards do we want to consider that a display is “HiDPI”. Probably would need some testing to see if e.g. scaling at `1.5` would set this option to `false`, but I think it looks safe to assume that anything equal or above `2.0` can be considered HiDPI.

Reply:
No, it's actually `>= 2.`

Reply:
I've re-read the commit log and the discussion, and indeed it should be `>= 2` 👌🏼